### PR TITLE
Covid Tracking App Capstone Review

### PR DESCRIPTION
**Pull request by Felipeg005**
In order to achieve project's requirements, next actions were taken:

## Home page
- [x] When the page loads, the webapp shows the list of countries to be filtered
- [x] Along with the category name, it displays the number of cases that come from the API.
- [x] When the user clicks (or taps) on a category item, the application navigates to the details page.

## Details page
- [x] In the details page, the webapp retrieves data from the API to show detailed data in the category.
- [x] When the user clicks on the "Back" button (<), the user navigates to the home page.

## Testing requirements
- [x] Unit tests for pure functions in Reducer code.
- [x] Create integration tests using the React Testing Library.
- [x] Mocks the access to the API, so tests don't send actual requests.
- [x] Mock the connection to the Redux Store.

## Technical requirements
- [x] The project is a single-page application (SPA) built with React and Redux.
- [x] The data retrieved from the API is stored in the Redux store.
- [x] Every page (the main page and the pages for each item) has a unique route within the SPA.

Here you can find the links for the features pulls that were merged in the development branch:

-[Initial Structure](https://github.com/Felipeg005/Covid19-Tracking-App/pull/1)
-[Feature/get API ](https://github.com/Felipeg005/Covid19-Tracking-App/pull/2)
-[Feature/display details ](https://github.com/Felipeg005/Covid19-Tracking-App/pull/3)
-[Feature/unit testing](https://github.com/Felipeg005/Covid19-Tracking-App/pull/4)

Please remember this project is to be reviewed in responsive mode for mobile.

- [x] The project was deployed and accessible online and[ you can find the link here](https://felipeg005.github.io/Covid19-Tracking-App/) and on the Readme file.

- [x] [And here's the link for video presentation](https://www.loom.com/share/daf5dc45e3094592816d7bb5b0bf884a) 

Thanks for your review 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,83 @@
-LOGO FAVICON <div>Iconos diseñados por <a href="https://www.flaticon.es/autores/kosonicon" title="kosonicon">kosonicon</a> from <a href="https://www.flaticon.es/" title="Flaticon">www.flaticon.es</a></div>
+## Covid-19 Tracker App
+
+![screenshot](./app_screenshot.png)
+
+## Live Demo
```
Comment (oloomoses) [inline README.md]: - [x] Kindly add a description of the project so as to make the readme more descriptive.

File: README.md [hunk 2]
```diff
@@ -1 +1,83 @@
...
+
+- Gh-Pages: [Visit Live Demo Link](https://felipeg005.github.io/Covid19-Tracking-App/)
+
+## Getting Started
+
+You can visit the Live Demo or clone the repository and open the dist/html.index file with your browser to see the project.
```
Comment (oloomoses) [inline README.md:17]: - [x] Kindly do a video presentation of the project highlighting all the project features and a piece of code you are proud of. Remember to add the video presentation link in the readme as per the project requirements.

File: README.md [hunk 3]
```diff
@@ -1 +1,83 @@
...
+
+## Acknowledgments
+
+- Thanks to Microverse for making it possible!
+
+- Nelson Sakwa creator of the wireframe on Behance.
+
+- LOGO FAVICON <div>Iconos diseñados por <a href="https://www.flaticon.es/autores/kosonicon" title="kosonicon">kosonicon</a> from <a href="https://www.flaticon.es/" title="Flaticon">www.flaticon.es</a></div>
+
+- <div>Icons made by <a href="https://www.flaticon.com/authors/konkapp" title="Konkapp">Konkapp</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></div>
```
Comment (oloomoses) [inline README.md:79]: Great job on the acknowledgments.

File: src/components/Nav.js [hunk 4]
```diff
@@ -1,26 +1,30 @@
 /* eslint-disable */
 import React from 'react';
```
Comment (oloomoses) [inline src/components/Nav.js]: - [x] The linters are there to enforce code quality and should not be disabled. Kindly enable the linters and fix the issues raised.

File: src/components/Country-lists.js [hunk 5]
```diff
@@ -12,10 +13,27 @@ const CountryLists = () => {
...
+        <li><h2 className="world-title">Covid-19<br/>Total World Cases</h2></li>
+        <li><h3 className="world-cases">{sumWorldCases()}</h3></li>
+        </div>
+      </ul>
+    </header>
     <main>
-    <h3>Stats by Country</h3>
+    <h3 className='stats-title'>Stats by Country</h3>
       <ul className="country-lists-container">
         {
```
Comment (oloomoses) [inline src/components/Country-lists.js]: - [x] Kindly create a filter component to allow a user to filter data with some parameters as per the project requirements.
Screenshot from 2021-09-09 20-23-55

- [x] The filter should be a stateless component
Screenshot from 2021-09-09 20-23-02

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @Felipeg005, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Felipeg005) [reply README.md]: Of course, thank you for the reminder. 😄

File: README.md (hunk 3 above)
Comment (Felipeg005) [reply README.md:79]: Thanks 🙂 👍

File: src/components/Nav.js (hunk 4 above)
Comment (Felipeg005) [reply src/components/Nav.js]: Totally right, I forgot to remove this disabled comment, now linters are enabled and without errors. 👍

File: README.md (hunk 2 above)
Comment (Felipeg005) [reply README.md:17]: 😟 sorry!, I forgot, Now I will do the video and update the pull and Readme file. 😄

File: src/components/Country-lists.js (hunk 5 above)
Comment (Felipeg005) [reply src/components/Country-lists.js]: Great, thanks for the advice, I really miss this requirement, now everything it's up to date, Thank you so much for the review 😄👍

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**